src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -129,7 +125,45 @@ export class HuiThermostatCard extends LitElement implements LovelaceCard {
             class="more-info"
             @click="${this._handleMoreInfo}"
           ></paper-icon-button>
-          <div id="thermostat"></div>
+          <div id="thermostat">
+            ${stateObj.state === "unavailable"
+              ? html`
+                  <round-slider
+                    .radius=${this._radius}
+                    disabled="true"
+                  ></round-slider>
+                `
+              : stateObj.attributes.target_temp_low &&
+                stateObj.attributes.target_temp_high
+              ? html`
+                  <round-slider
+                    .radius=${this._radius}
+                    .low=${stateObj.attributes.target_temp_low}
+                    .high=${stateObj.attributes.target_temp_high}
+                    .min=${stateObj.attributes.min_temp}
+                    .max=${stateObj.attributes.max_temp}
+                    .step=${this._stepSize}
+                    @value-changing=${this._dragEvent}
+                    @value-changed=${this._setTemperature}
+                    handleSize="10"
```

Comment:
Can we make the handleSize the same as the light card? This feels to big, especially since they get even bigger on focus.